Formula/luajit.rb

```diff
@@ -1,21 +1,38 @@
+# NOTE: We have a policy of building only from tagged commits, but make a
+#       singular exception for luajit. This exception will not be extended
+#       to other formulae. See:
+#       https://github.com/Homebrew/homebrew-core/pull/99580
 class Luajit < Formula
   desc "Just-In-Time Compiler (JIT) for the Lua programming language"
   homepage "https://luajit.org/luajit.html"
+  # Update this to the tip of the `v2.1` branch at the start of every month.
+  url "https://github.com/LuaJIT/LuaJIT/archive/50936d784474747b4569d988767f1b5bab8bb6d0.tar.gz"
+  # Use the version scheme `2.1.0-beta3-yyyymmdd.x` where `yyyymmdd` is the date of the
+  # latest commit at the time of updating, and `x` is the number of commits on that date.
+  version "2.1.0-beta3-20220712.6"
```

Comment:
How do we know if it's beta3/4/5? 

Random thought that might be a little more work here: perhaps we could use `git describe --all --long --first-parent` output for this, or something similar?